src/main/cpp/Shooter.cpp

```diff
@@ -163,109 +166,107 @@ Shooter::Aim(){
             angle2 = data2.first;
             speed2 = data2.second;
 
-            m_angle = (angle1 + angle2)/2 * angle_scale_factor;
-            m_speed = (speed1 + speed2)/2 * speed_scale_factor;
-            //frc::SmartDashboard::PutNumber("velocity", m_speed);
-            //frc::SmartDashboard::PutNumber("position",  m_angle);
+            m_angle = (angle1 + angle2) / 2 * angle_scale_factor;
+            m_speed = (speed1 + speed2) / 2 * speed_scale_factor;
+            // frc::SmartDashboard::PutNumber("velocity", m_speed);
+            // frc::SmartDashboard::PutNumber("position",  m_angle);
 
-            //Sets last known X, Y, and distance
+            // Sets last known X, Y, and distance
             m_lastX = m_robotX;
             m_lastY = m_robotY;
             m_lastDistance = point;
 
-        }else{
+        } else {
             m_angle = 0;
             m_speed = 0;
         }
     }
     // Set Turret movement
-    double x_off = m_limelight->getXOff()+3.85;
-    if(x_off == 10000.0 && m_OdometryErrorRate < 10){
-        //tan of distance between hub and robot
-        double hubX = m_lastDistance*cos(m_lastAngle/PI*180)

```

Comment:
I had to change this area because it didn't compile before, no semicolons and mismatched parens.